source/core_http_client.c

```diff
@@ -924,12 +929,8 @@ static int httpParserOnBodyCallback( llhttp_t * pHttpParser,
 
     /* The next location to write. */
 
-    /* MISRA Rule 11.8 flags casting away the const qualifier in the pointer
-     * type. This rule is suppressed because when the body is of transfer
-     * encoding chunked, the body must be copied over the chunk headers that
-     * precede it. This is done to have a contiguous response body. This does
-     * affect future parsing as the changed segment will always be before the
-     * next place to parse. */
+    /* MISRA Ref 11.8.1 [Function pointer and use of const pointer] */
```

Comment:
This does not involve a function pointer.

Reply:
This one is about the const pointer. The variable [pBody](https://github.com/FreeRTOS/coreHTTP/blob/main/source/include/core_http_client.h#L486) is declared as a const pointer, so dropping the cast here to drop the const sets off the rule.

Reply:
Yes, so can we remove function pointer from the title for better readability?

Reply:
Sorry, misunderstood your request. I've changed the title to "Removal of const from pointer"